scripts/irods/lib.py

```diff
@@ -465,7 +467,12 @@ def default_prompt(*args, **kwargs):
                     args[0] % tuple(args[1:]), ':\n',
                     '\n'.join(['%d. %s' % (i + 1, default[i]) for i in range(0, len(default))]),
                     '\nPlease select a number or choose 0 to enter a new value'])
-                user_input = default_prompt(message, default=[1], **kwargs)
+
+                if previous:
+                    user_input = default_prompt(message, default=[previous], **kwargs)
+                else:
+                    user_input = default_prompt(message, default=[1], **kwargs)
+
```

Comment:
An alternative...
```python
user_input = default_prompt(message, default=[previous if previous else 1], **kwargs)
```

Reply:
Shiny!

Reply:
well that looks fun.    will update once back.

Reply:
done